mava/configs/env/cleaner.yaml

```diff
@@ -13,6 +13,9 @@ eval_metric: episode_return
 # Whether the environment observations encode implicit agent IDs. If True, the AgentID wrapper is not used.
 # This should not be changed.
 implicit_agent_id: True
+# Whether or not to log the winrate of this environment. This should not be changed as not all
+# environments have a winrate metric.
+log_win_rate: True
```

Comment:
NIT: can we move them up next to the eval_metric for better visibility 🙏

Reply:
I put it here because both `log_win_rate` and `implicit_agent_id` shouldn't ever be changed I think, so we don't want to it to be too high up because we shouldn't be changing it, what do you think?